### PR TITLE
V2 webhook updates

### PR DESCRIPTION
**Pull request by keithmacaulay**
- Bumping benchling-sdk to v1.13.0 for updated webhook models
- Updating handler.py to use V2 webhooks ahead of V0 deprecation (https://docs.benchling.com/changelog/explicit-webhook-subscriptions-and-webhook-version-update)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,5 @@
 **/.client_secret
 **/.pytest_cache
 .mypy_cache
-.ruff_cache
\ No newline at end of file
+.ruff_cache
+**/.nvm/
```
Comment (rweinberger) [inline .gitignore:12]: not sure we should be modifying the `.gitignore` for a change like this, why was this line needed?

**Review by rweinberger — CHANGES_REQUESTED** [review]
Code changes look great, tested it locally and works fine! Just one more thing to add into this PR: we'll soon be releasing the ability to explicitly subscribe to specific webhooks via the manifest. In order to be forwards-compatible with that world, we should augment the `manifest.yaml` file with the following `subscriptions` section that includes the two webhooks that this app needs in order to function properly:

```
subscriptions:
  deliveryMethod: WEBHOOK
  messages:
  - type: v2.canvas.initialized
  - type: v2.canvas.userInteracted
```
For now, we're still sending all webhooks to existing apps, but we won't be doing that forever. So if we don't make this manifest change to opt in to receiving these webhooks, future example apps created from the manifest will actually no longer receive these webhooks.

File: .gitignore (hunk 1 above)
Comment (keithmacaulay) [reply .gitignore]: Good catch on the manifest, updated. Reverted the .gitignore updates.

**Review by rweinberger — APPROVED** [review]
ship it!! 🚢 🚢  and thank you so much for taking the initiative to put this PR up 🎉
